faq.rst

```diff
@@ -37,7 +38,7 @@ Type
 **QUESTION**: What behavior is expected from a purl spec implementation if a
 ``type`` contains a character like a slash '/' or a colon ':'?
 
-The "Rules for each purl component" section provides that
+**ANSWER**: The "Rules for each purl component" section provides that
 
     [t]he package ``type`` MUST be composed only of ASCII letters and numbers,
```

Comment:
@johnmhoran Can we refine this with the new wording? and remove the the weird square brackets in `[t]`he?

Reply:
@pombredanne I've fixed the use of square brackets (thanks for catching that) and will commit and push these updates.  I'm not sure what you are referring to by "the new wording" aside from the square brackets -- please clarify as needed once the revised `faq.rst` has been pushed.